sdcm/cluster.py

```diff
@@ -4705,15 +4705,17 @@ def kill_cassandra_stress_thread(self):
         for loader in self.nodes:
             try:
                 loader.remoter.run(cmd='pgrep -f cassandra-stress | xargs -I{}  kill -TERM -{}',
-                                   verbose=False, ignore_status=True)
+                                   verbose=False, ignore_status=False)
```

Comment:
@fruch I switched `ignore_status` to `False` - so in case it fails try-catch will raise warning.
Is my thinking correct? As I'm confused why it wasn't `False` from the beginning.

Reply:
I think it's before there was a try/expect there.

Reply:
looks this throws exception always - regardless if processes got found/killed. I'll remove this change as it doesn't introduce anything valuable. We need to think how to properly implement this.